spec/models/manageiq/providers/embedded_ansible/automation_manager/configuration_script_source_spec.rb

```diff
@@ -26,6 +27,7 @@
 
       GitRepository
       stub_const("GitRepository::GIT_REPO_DIRECTORY", repo_dir)
+      stub_const("GitRepository::LOCKFILE_DIR", locks_dir)
```

Comment:
NOTE unlike the git_repository_spec there isn't always a repos instance available (this also tests creating via the create_in_provider_queue method for example), since this already stubs the GIT_REPO_DIRECTORY so that it can be cleaned up later this felt like another case of the same issue.